lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -180,8 +181,8 @@ if Code.ensure_loaded?(Membrane.WebRTC.EndpointBin) do
     end
 
     @impl true
-    def handle_notification({:vad, val}, {:endpoint, endpoint_id}, _ctx, state) do
-      send(state.owner, {:vad_notification, val, endpoint_id})
+    def handle_notification({:vad, val}, :endpoint_bin, _ctx, state) do
+      send(state.owner, {:vad_notification, val, state.ice_name})
```

Comment:
Let's use name from ctx https://hexdocs.pm/membrane_core/Membrane.Bin.CallbackContext.Other.html
Using `state.ice_name`  is counter intuitve as `ice_name` suggests relation with ICE protocol